### ISSUE TITLE
playback of mp2 audio files?

### ISSUE THREAD

**Issue by kookster** [phoboslab/jsmpeg#123]
So cool. We have places where we play back just mp2 audio files, is there a way to use this for that purpose?

**Comment by phoboslab** [comment]
Absolutely. Just repack your mp2 in an mpegts container (`ffmpeg -i in.mp2 -f mpegts -c:a copy out.ts`) and set `video: false` in the JSMpeg options.
